_posts/en/newsletters/2023-08-02-newsletter.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Bitcoin Optech Newsletter #262'
+permalink: /en/newsletters/2023/08/02/
+name: 2023-08-02-newsletter
+slug: 2023-08-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Transcripts of periodic LN specification meetings:** Carla
+  Kirk-Cohen [posted][kc scripts] to the Lightning-Dev mailing list to
+  announce that the last several video conference meetings to discuss
+  changes to the LN specification were transcribed.  The transcripts are
+  now [available][btcscripts spec] on Bitcoin Transcripts.  In related
+  news, as discussed a few weeks ago during the in-person LN developer
+  conference, the '#lightning-dev` IRC chatroom on the [Libera.chat][]
+  network has seen a significant burst of renewed activity for
+  LN-related discussion.
+
+- **Safety of blind MuSig2 signing:** Tom Trevethan [posted][trevethan
+  blind] to the Bitcoin-Dev mailing list to request a review of a
+  cryptographic protocol planned as part of a [statechains][topic
+  statechains] deployment.  The goal was to deploy a service that would
+  use its private key to create a [MuSig2][topic musig] partial
+  signature without gaining any knowledge about what it was signing or
+  how its partial signature was used.  The blind signer would simply
+  report how many signatures it had created with a particular key.
+
+    Discussion on the list examined pitfalls of various constructions
+    related to the specific problem and to [even more generalized blind
```

Comment:
```suggestion
    related to the specific problem and of [even more generalized blind
```
But I'm not sure if I'm parsing this sentence correctly. The discussion examined generalized blind schnorr signing pitfalls? If so, I think my suggestion is preferred.